packages/manager/apps/exchange/webpack.config.js

```diff
@@ -44,8 +62,16 @@ module.exports = (env = {}) => {
       mainFields: ['module', 'browser', 'main'],
     },
     plugins: [
+      new webpack.ContextReplacementPlugin(
+        /moment[/\\]locale$/,
+        /de|en-gb|es|es-us|fr-ca|fr|it|pl|pt/,
+      ),
+
       new webpack.DefinePlugin({
-        __WEBPACK_REGION__: `'${REGION}'`,
+        __NODE_ENV__: process.env.NODE_ENV
+          ? `'${process.env.NODE_ENV}'`
+          : '"development"',
+        __NG_APP_INJECTIONS__: getNgAppInjections(['EU', 'CA', 'US']),
```

Comment:
```suggestion
        __NG_APP_INJECTIONS__: getNgAppInjections(['EU', 'CA']),
```
US needed for Exchange ?

Reply:
nope, It's a bad copy/paste. Was done for all apps, so I have to update and check REGION availability (telecom apps...)

Reply:
I think we could keep it like this. It's not the app build to decide if the product is enabled or not for `US` region.

Reply:
@cbourgois Ok but in that cas should we specify a region then ? If it's not the app build responsibility we should be region agnostic, right ?

Reply:
Yes, we have to refacto `@ovh-ux/manager-webpack-config` to have all of this in it. Should be done in another PR. Here the goal is to kill Environment singleton.